Shifty.App/DomainModels/MenuItem.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Data.Common;
+using Shifty.Api.Generated.AnalogCoreV1;
+using Shifty.Api.Generated.AnalogCoreV2;
+
+namespace Shifty.App.DomainModels
+{
+    public class MenuItem {
+        public int Id { get; init; }
+        public string Name { get; set; }
+        public static MenuItem FromDto(MenuItemResponse dto)
+        {
+            return new MenuItem()
+            {
+                Id = dto.Id,
+                Name = dto.Name
+            };
+        }
+
+        public static AddMenuItemRequest ToAddRequest(MenuItem menuItem)
+        {
+            return new AddMenuItemRequest()
+            {
+                Name = menuItem.Name
+            };
+        }
+
+        public static UpdateMenuItemRequest ToUpdateRequest(MenuItem menuItem)
+        {
+            return new UpdateMenuItemRequest()
+            {
+                Name = menuItem.Name
+            };
+        }
+
+        public override string ToString()
+        {
+            return Name;
+        }
```

Comment:
This is in no way trying to blame others - but according to my GitLens, this stems from @fredpetersen but looking at the GitHub commit, it seems to be a commit that we both authored so I cannot figure out if this is me or @fredpetersen 's code. I couldn't see any issues when removing it (both for menuitem and products). I assume it might be outdated code from one of my attempts at having a good dropdown solution which ended up using the MudExtensions ComboBox.

I have removed the `ToString()` method on these two domain model objects, but if they are required for the Menu Item Manager then please reintroduce.